src/inflation/adapter/load_files.clj

```diff
@@ -0,0 +1,38 @@
+(ns inflation.adapter.load_files
+  (:require
+   [clojure.string :as str]
+   [datomic.api :as d]
+   [inflation.db.datomic.db :as db]
+   [inflation.model.price :as model]
+   [clojure.data.csv :as csv]
+   [clojure.java.io :as io]))
+
+(def conn (db/connection))
+
+(db/create-schema conn)
+
+(defn insert-entity [entity]
+  (prn entity)
+  (d/transact conn [entity]))
+
+(defn read-file [filename] 
+  (with-open [reader (io/reader filename)]
+    (doall
+     (csv/read-csv reader))))
+
+(defn create-price [item, country, type]
+  (let [value (bigdec (nth item 2))
+        month (long (Float/valueOf (nth item 1)))
+        year (long (Float/valueOf (nth item 0)))]
+    (model/new-price country type value year month)))
```

Comment:
This is not only for clojure, but avoid having functions with several positional parameters. Those functions are hard to extend when you need new data. Prefer using a map instead.